src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,17 +13,60 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    private static final String HELP_INTRO = "Below are some basic instructions to get started using PROperty!";
```

Comment:
I just implemented a fing tag command. Can implement if you want but next iteration also no issue.

Reply:
Alright! Will get to it.

Reply:
Added